### ISSUE TITLE
Setup pseudo-labelling pipeline for Financial PhraseBank

### ISSUE THREAD

**Issue by Bikram2051** [Kevinmonitor/TECHNOLOGY-DESIGN-PROJECT#7]
Load pre-trained GoEmotions emotion classifier (SamLowe/roberta-base-go_emotions) to pseudo-label FPB sentences with 6-class emotion labels.

## Tasks
- [x] Install transformers to load classifier
- [x] Install PyTorch to set up pipeline and running the emotion classifier
- [x] Transformers upgraded for compatibility
- [x] Loaded pre-trained GoEmotions emotion classifier and tested on sample sentence
- [x] Pseudo-label Financial PhraseBank with 6 emotion classes
- [ ] Validate pseudo-labels against original FPB labels (cross-tabulation heatmap)
- [ ] Compare 6-class distributions across both datasets
- [ ] Inspect sample pseudo-labelled sentences
- [ ] Export unified CSVs for team model training
